db/migrate/20250710181656_copy_ai_helper_model_to_persona_default.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class CopyAiHelperModelToPersonaDefault < ActiveRecord::Migration[7.2]
+  def up
+    ai_helper_model =
+      DB.query_single("SELECT value FROM site_settings WHERE name = 'ai_helper_model'").first
+
+    if ai_helper_model.present? && ai_helper_model.start_with?("custom:")
+      # Extract the model ID from the setting value (e.g., "custom:1" -> "1")
+      model_id = ai_helper_model.split(":").last
+
+      # Update the helper personas with the extracted model ID
+      execute(<<~SQL)
+        UPDATE ai_personas
+        SET default_llm_id = #{model_id}
+        WHERE id IN (-18, -19, -20, -21, -22, -23, -24, -25) AND default_llm_id IS NULL
```

Comment:
We need to make sure this also works for sites using custom personas by adding the IDs from:

- ai_helper_proofreader_persona
- ai_helper_title_suggestions_persona
- ai_helper_explain_persona
- ai_helper_post_illustrator_persona
- ai_helper_smart_dates_persona
- ai_helper_translator_persona
- ai_helper_markdown_tables_persona
- ai_helper_custom_prompt_persona

There are some models that rely on vision here. I saw there's another migration for image caption, but do we need to use the image caption model for those too?